docs/development.md

```diff
@@ -153,18 +153,18 @@ export AZURE_CLIENT_SECRET_B64="$(echo -n "$AZURE_CLIENT_SECRET" | base64 | tr -
 
 # Machine settings.
 export CONTROL_PLANE_MACHINE_COUNT=3
-export CONTROL_PLANE_MACHINE_TYPE="Standard_D2s_v3"
-export NODE_MACHINE_TYPE="Standard_D2s_v3"
+export AZURE_CONTROL_PLANE_MACHINE_TYPE="Standard_D2s_v3"
+export AZURE_NODE_MACHINE_TYPE="Standard_D2s_v3"
 export WORKER_MACHINE_COUNT=2
 export KUBERNETES_VERSION="1.16.7"
 
 # Generate SSH key.
-# If you want to provide your own key, skip this step and set SSH_PUBLIC_KEY to your existing file.
+# If you want to provide your own key, skip this step and set AZURE_SSH_PUBLIC_KEY to your existing file.
 SSH_KEY_FILE=.sshkey
 rm -f "${SSH_KEY_FILE}" 2>/dev/null
 ssh-keygen -t rsa -b 2048 -f "${SSH_KEY_FILE}" -N '' 1>/dev/null
 echo "Machine SSH key generated in ${SSH_KEY_FILE}"
-export SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\r\n')
+export AZURE_SSH_PUBLIC_KEY=$(cat "${SSH_KEY_FILE}.pub" | base64 | tr -d '\r\n')
```

Comment:
Shouldnt this be the same across all providers? Seems like WORKER_MACHINE_COUNT and KUBERNETES_VERSION remains intact.

Reply:
I assumed you could have different keys across providers